main.tf

```diff
@@ -168,28 +168,22 @@ locals {
       for pol_key, pol_val in lookup(entry_val, "policy_associations", {}) :
       merge(
         {
-          principal_arn     = entry_val.principal_arn
-          kubernetes_groups = lookup(entry_val, "kubernetes_groups", [])
```

Comment:
`kubernetes_groups`, `tags`, `type`, and `user_name` are used in the access entry, not in the policy association. So these are not used here and can be removed